src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,45 @@
+package duke.command;
+
+import duke.Storage;
+import duke.Ui;
+import duke.task.TaskList;
+
+/**
+ * Represents the command for the deleting tasks from Duke's TaskList.
+ */
+public class DeleteCommand implements Command{
+
+    private final int toDelete;
```

Comment:
Perhaps `toDelete` should be capitalised since it is a constant? [(Reference: Constant names must be all uppercase using underscore to separate words)](https://se-education.org/guides/conventions/java/index.html#naming)

```suggestion
    private final int TO_DELETE;
```